packages/proxy/src/proxy.ts

```diff
@@ -766,7 +766,7 @@ async function fetchModelLoop(
 
     const modelSpec =
       (model !== null
-        ? (secret.metadata?.customModels?.[model] ?? AvailableModels[model])
+        ? secret.metadata?.customModels?.[model] ?? AvailableModels[model]
```

Comment:
I'm not yet sure why in CI this change was necessary but locally (with presumably the same prettier version) it prefers with `()`